README.md

```diff
@@ -1,12 +1,14 @@
-# Node Starter App
-
-Quick start application setup with sensible defaults baked in.
+# Canada Web Forms Starter Repo
 
 **Demo:** https://cds-node-starter.herokuapp.com
 
-Should you use this? It depends ;)
+This repository provides a codebase that can be used to quickly start an application for building web-based forms that look like a Government of Canada website. It's setup with some sensible defaults and tech choices, such as:
```

Comment:
Note: It could just be a page with a few views and no forms so might want to reword this.

Reply:
Good suggestion! Rephrased. How does it look now?